Vagrantfile

```diff
@@ -207,7 +253,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     if Vagrant.has_plugin?("vagrant-vbguest")
         config.vbguest.auto_update = false
     end 
-    if ENV['CONTIV_NODE_OS'] && ENV['CONTIV_NODE_OS'] == "ubuntu" then
+    if node_os == "ubuntu" then
```

Comment:
above, syntax used is #{node_os}. Is there a reason for a different syntax here?